app/models/Integration.ts

```diff
@@ -31,6 +31,10 @@ class Integration<T = unknown> extends Model {
 
   @observable
   settings: IntegrationSettings<T>;
+
+  accessToken?: string;
+
+  refreshToken?: string;
```

Comment:
```suggestion
```

This change can be reverted, right?

Reply:
Let me take a look at this PR this week.. I think there are some unfinished things here from the feedback in #7461..

Reply:
Taking a while to get around to this.. I'll circle back here after finishing the current commitments.

Reply:
roger that